services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -162,6 +190,31 @@ export const SubmissionSummary = (): React.ReactElement => {
     // Only show for CMS_USER or CMS_APPROVER_USER users
     const showSubmissionApproval = submissionApprovalFlag && hasCMSPermissions
 
+    const approveContractAction = async (actionModalInput?: string) => {
+        setIsSubmitting(true)
```

Comment:
We should log this submit event to Tealium. You could do this like `UnlockSubmitModal` does with `logFormSubmitEvent`